docs/source/notebooks/dap_subset.ipynb

```diff
@@ -946,7 +948,7 @@
+       "    NCO:            4.0.9</pre><div class='xr-wrap' hidden><div class='xr-header'><div class='xr-obj-type'>xarray.Dataset</div></div><ul class='xr-sections'><li class='xr-section-item'><input id='section-458c8422-74c9-4e0e-858f-5f40e5ed2446' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-458c8422-74c9-4e0e-858f-5f40e5ed2446' class='xr-section-summary'  title='Expand/collapse section'>Dimensions:</label><div class='xr-section-inline-details'><ul class='xr-dim-list'><li><span>lat</span>: 6</li><li><span>lon</span>: 7</li><li><span>time</span>: 1</li></ul></div><div class='xr-section-details'></div></li><li class='xr-section-item'><input id='section-f129854c-3a1a-46b3-8984-9930e4f00563' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-f129854c-3a1a-46b3-8984-9930e4f00563' class='xr-section-summary'  title='Expand/collapse section'>Coordinates: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'></ul></div></li><li class='xr-section-item'><input id='section-8c0701a4-5dce-4c6a-94c9-56730df87378' class='xr-section-summary-in' type='checkbox'  checked><label for='section-8c0701a4-5dce-4c6a-94c9-56730df87378' class='xr-section-summary' >Data variables: <span>(1)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'><li class='xr-var-item'><div class='xr-var-name'><span>pr</span></div><div class='xr-var-dims'>(time, lat, lon)</div><div class='xr-var-dtype'>float32</div><div class='xr-var-preview xr-preview'>...</div><input id='attrs-cbcb535a-0d95-4040-912a-9092a9a9de11' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-cbcb535a-0d95-4040-912a-9092a9a9de11' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-458c54ce-7904-4afe-8a4e-d822ad4b6d22' class='xr-var-data-in' type='checkbox'><label for='data-458c54ce-7904-4afe-8a4e-d822ad4b6d22' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>original_name :</span></dt><dd>APRL+APRC</dd><dt><span>long_name :</span></dt><dd>Precipitation</dd><dt><span>standard_name :</span></dt><dd>precipitation_flux</dd><dt><span>cell_methods :</span></dt><dd>time: mean (interval: 30 minutes)</dd><dt><span>units :</span></dt><dd>kg m-2 s-1</dd><dt><span>history :</span></dt><dd>pr=max(0,pr) applied to raw data;</dd><dt><span>_ChunkSizes :</span></dt><dd>[7200    6    7]</dd></dl></div><div class='xr-var-data'><pre>array([[[3.415346e-05, 2.171099e-05, 1.613796e-05, 4.132092e-05,\n",
```

Comment:
I still see html here, I am a bit surprised you say xarray rendering reverted back to text.

Reply:
When I open the notebook, I get text output, although when I ran the notebook manually, I got html...